semantic_conventions.md

```diff
@@ -18,6 +18,7 @@ Span tags apply to **the entire Span**; as such, they apply to the entire timera
 |:--------------|:-----|:-------------------|
 | `component` | string  | The software package, framework, library, or module that generated the associated Span. E.g., `"grpc"`, `"django"`, `"JDBI"`. |
 | `db.instance` | string | Database instance name. E.g., In java, if the jdbc.url=`"jdbc:mysql://127.0.0.1:3306/customers"`, the instance name is `"customers"`. |
+| `db.collection` | string | Database collection name, generally for nosql type databases. E.g., for MongoDB this is the collection. |
```

Comment:
Could you please provide more examples for different databases so we know we are not introducing a tag only for mongo?

Reply:
Enough to put it here as a comment